_config.yml

```diff
@@ -31,6 +31,12 @@ collections:
   districts:
     output: true
     permalink: '/district/:path/'
+  term_9_posts:
+    output: true
+    permalink: '/term/9/post/:path/'
+  term_10_posts:
+    output: true
+    permalink: '/term/10/post/:path/'
```

Comment:
I think the post is the more important thing here than the term — i.e. the URL should be more like `/post/whatever/9`, `/post/whatever/10` etc.